plugins/src/ios/InitializePluginManager.m

```diff
@@ -0,0 +1,27 @@
+//
+//  InitializePluginManager.m
+//  Example_App
+//
+//  Created by Barak Ben Hur on 03/08/2022.
+//
+
+#import "InitializePluginManager.h"
+#import "Example_App-Swift.h"
```

Comment:
Don't think we want to import this header

Reply:
#import "Example_App-Swift.h" removed

Reply:
This this is back :)

Reply:
sorry copied it from a backup :)

Reply:
fixed